spec/acceptance/nodesets/ubuntu-16-docker.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  ubuntu-16-docker:
+    roles:
+      - master
+    platform: ubuntu-1404-x86_64
```

Comment:
As discussed before, Beaker + Ubuntu 16.04 doesn't work out of the box. I tracked down the [exact bug report for the issue we're hitting](https://tickets.puppetlabs.com/browse/BKR-821). Also, see this more generic [add-Ubuntu 1604-support bug](https://tickets.puppetlabs.com/browse/BKR-708).